### PR TITLE
feb dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-february-2022.md [hunk 1]
```diff
@@ -0,0 +1,286 @@
+Title: iRODS Development Update: February 2022
+Date: 2022-02-22 12:00
```
Comment (trel) [inline content/posts/irods-development-update-february-2022.md:2]: could be today 3pm if you want

File: content/posts/irods-development-update-february-2022.md [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+Major work on the project structure is also underway. Doing this will help in providing better CMake support, understanding of how components fit together, and help new developers contribute to the project.
+
+Given that many workloads involve the delay server, we hope people are excited to hear that work on allowing migration of the delay server is partially complete. Changes to the setup script and database are in-place. The final piece of the puzzle involves implementing the leader/successor algorithm.
+
+The Zone Management Tool has grown the ability to manage passwords and support for case-insensitive search. These changes are powered by enhancements to the C++ REST API. Not only that, but more health checks centered around resources have been added.
+
+Notable releases this month include the Python iRODS CLient 1.1.1, Python Rule Engine Plugin 4.2.11.1, Globus Connector 4.2.11.1, NFSRODS 2.1.0, C++ REST API 0.9.0, and Jargon 4.3.2.4-RELEASE
```
Comment (trel) [inline content/posts/irods-development-update-february-2022.md:24]: "CLient" -> "Client"

needs trailing period

File: content/posts/irods-development-update-february-2022.md [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+
+    - NetCDF microservices
+        - released 4.2.9.0, 4.2.10.0, and 4.2.11.0
+        - full HDF5 support could come later
+
+- Discussion
+
+    - GenQuery should use a syntax closer to SQL
+        - https://github.com/irods/irods/issues/6168
+
```
Comment (trel) [inline content/posts/irods-development-update-february-2022.md:286]: - Grouper integration / syncing

Comment (korydraughn) [reply content/posts/irods-development-update-february-2022.md:286]: good catch

File: content/posts/irods-development-update-february-2022.md (hunk 1 above)
Comment (korydraughn) [reply content/posts/irods-development-update-february-2022.md:2]: sure! will change to 2022-02-21 15:00

Comment (korydraughn) [reply content/posts/irods-development-update-february-2022.md]: done

File: content/posts/irods-development-update-february-2022.md (hunk 2 above)
Comment (korydraughn) [reply content/posts/irods-development-update-february-2022.md]: done

File: content/posts/irods-development-update-february-2022.md (hunk 3 above)
Comment (korydraughn) [reply content/posts/irods-development-update-february-2022.md:286]: done

**Pull request merged**